docs/api/memory-constraints.md

```diff
@@ -0,0 +1,21 @@
+# OneBlink API CLI
+
+## Memory Constraints
+
+By default Memory for your api is limited to 1024 MB (1 GB).
+
+### Increasing Memory Size
+
+We allow for increasing the Memory Size of your api through the `server.memorySize` property in the `.blinkmrc.json` file;
+
+#### .blinkmrc.json
+
+The following configuration will allow 2GB for your api functions:
+
+```json
+{
+  "server": {
+    "memorySize": 2048
```

Comment:
I expect aws only allows a certain number of values here, we should document what they are.

Reply:
Yeh, multiples of 64, Ill put it in the doc